R/robustbayesianmetaanalysis.R

```diff
@@ -1171,52 +1175,52 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     type     = "models",
     logBF    = options[["bayesFactorType"]] == "LogBF10",
     BF01     = options[["bayesFactorType"]] == "BF01",
-    probs    = c(.5 + c(-1, 1) * options[["results_CI"]] / 2)
+    probs    = c(.5 + c(-1, 1) * options[["resultsCI"]] / 2)
   )
   
   # do ordering
-  if (options[["results_models_order"]] == "marglik") {
+  if (options[["resultsModelsOrder"]] == "marglik") {
     fitSummary[["overview"]] <- fitSummary[["overview"]][order(fitSummary[["overview"]][["log(MargLik)"]], decreasing = TRUE),]
-  } else if (options[["results_models_order"]] == "posterior") {
+  } else if (options[["resultsModelsOrder"]] == "posterior") {
     fitSummary[["overview"]] <- fitSummary[["overview"]][order(fitSummary[["overview"]][["Post. prob."]], decreasing = TRUE),]
   }
   
   # compute the BF requested
-  if (options[["results_models_BF"]] == "inclusion") {
+  if (options[["resultsModelsBF"]] == "inclusion") {
     bf <- fitSummary[["overview"]][, 7]
-  } else if (options[["results_models_BF"]] == "best") {
+  } else if (options[["resultsModelsBF"]] == "best") {
     bf <- exp(fitSummary[["overview"]][["log(MargLik)"]]) / exp(max(fitSummary[["overview"]][["log(MargLik)"]]))
-  } else if (options[["results_models_BF"]] == "previous") {
+  } else if (options[["resultsModelsBF"]] == "previous") {
     temp_this <- exp(fitSummary[["overview"]][["log(MargLik)"]])[-length(fitSummary[["overview"]][["log(MargLik)"]])]
```

Comment:
I'm not sure what this refers to temp_this and temp_prev

Reply:
to a marginal likelihood of the model corresponding to this row and marginal likelihood corresponding to the previous row.  Renamed to `tempThisMargLik` and `tempPrevMargLik`